CHANGES.md

```diff
@@ -22,6 +22,8 @@
   + Fix formatting of empty signature payload `[%a:]` (#1236) (Etienne Millon)
 
   + Fix parenthesizing when accessing field of construct application (#1247) (Guillaume Petiot)
+  
+  + Fix regression with `indicate-multiline-delimiters=space` (#1169) (Josh Berdine)
```

Comment:
@gpetiot You could add your name as you fixed big issues.

Reply:
done